pkg/internal/controller/controller.go

```diff
@@ -215,23 +218,26 @@ func (c *Controller) reconcileHandler(obj interface{}) bool {
 		c.updateMetrics(time.Since(reconcileStartTS))
 	}()
 
-	var req reconcile.Request
-	var ok bool
-	if req, ok = obj.(reconcile.Request); !ok {
+	// Declare a local log using this controller's name.
+	log := log.WithName(c.Name)
```

Comment:
Why do we have to initialize it twice, can we store it on the controller instead?

Reply:
Yeah that's a good point, changed it up.